www/include/TestResults.php

```diff
@@ -186,8 +186,17 @@ public function getMedianRunNumber($metric, $cached, $medianMode = "median") {
     if ($numValues == 1 || $medianMode == "fastest") {
       return $runNumbers[0];
     }
-    $medianIndex = (int)floor($numValues / 2.0);
-    return $runNumbers[$medianIndex];
+    $medianIndex = (int)floor(((float)$numValues + 1.0) / 2.0);
```

Comment:
Sure, I was just using the same code pattern as in getMedianRun() so there's consistency in the approach. Having two different ways of doing the same thing is what led to this issue. Ideally there should only be one function that gets the median run rather than two.